app/src/main/java/io/homeassistant/companion/android/webview/WebViewActivity.kt

```diff
@@ -459,22 +469,64 @@ class WebViewActivity : AppCompatActivity(), io.homeassistant.companion.android.
     }
 
     @SuppressLint("InflateParams")
-    override fun authenticationDialog(handler: HttpAuthHandler) {
+    fun authenticationDialog(handler: HttpAuthHandler, host: String, authError: Boolean) {
         val inflater = layoutInflater
         val dialogLayout = inflater.inflate(R.layout.dialog_authentication, null)
         val username = dialogLayout.findViewById<EditText>(R.id.username)
         val password = dialogLayout.findViewById<EditText>(R.id.password)
-
-        AlertDialog.Builder(this)
-            .setTitle(R.string.auth_request)
-            .setView(dialogLayout)
-            .setPositiveButton(android.R.string.ok) { _, _ ->
-                handler.proceed(username.text.toString(), password.text.toString())
+        val remember = dialogLayout.findViewById<CheckBox>(R.id.checkBox)
```

Comment:
Why is there a remember me checkbox? Isn't that the same as disabling auth completely?

Reply:
I assume this is not the same thing, a user can access a site or a resource in different ways, this application for example, so he will not necessarily want to disable authentication. On the other hand, if the application is already locked (by credential or biometric), he can use this "Remember" function so as not to have to re-authenticate himself on his different sites. It's a bit like a SSO.

Reply:
That makes no sense to me. You either have biometric authentication turned on, and so you are prompted. If you don't want it, you disable it. 

Let's remove this option.

Reply:
I am sorry for intervention, but as we have now confirmed that the authentication dialog is for Basic HTTP auth, shouldn't we keep remember me button? To me it really does make sense to have it. You wouldn't want to login to iFrame every time you open the app.

Reply:
@balloob this is for saving external authentication a panel or iFrame that requires basic auth.  This shouldn't have any bearing on the authentication to Home Assistant itself.